### PR TITLE
Make keyboard pop up automatically

### LINKED ISSUE #66: Automatically open keyboard when adding key location

### PR DESCRIPTION
**Pull request by kangp3**
Closes #66

### CODE REVIEW

File: app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java [hunk 1]
```diff
@@ -61,8 +62,20 @@ public void onClick(DialogInterface dialog, int which) {
...
+        AlertDialog d = (AlertDialog) getDialog();
+        d.getButton(AlertDialog.BUTTON_POSITIVE).setTextColor(Color.BLACK);
+        d.getButton(AlertDialog.BUTTON_NEGATIVE).setTextColor(Color.BLACK);
+
+        EditText e = (EditText) d.findViewById(R.id.key_loc_name_input);
+        e.setOnFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View v, boolean hasFocus) {
+                if (hasFocus) {
+                    AlertDialog d = (AlertDialog) getDialog();
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java]: Are you not able to use the `AlertDialog d` you defined outside of this function?

Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java]: Since it's an inner class, I'd have to define the higher one as `final`

Comment (kangp3) [inline app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java]: Fixed

**Comment by kangp3** [comment]
If you LGTM I will squash and re-push

File: app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java [hunk 2]
```diff
@@ -61,8 +62,19 @@ public void onClick(DialogInterface dialog, int which) {
...
+        final AlertDialog d = (AlertDialog) getDialog();
+        d.getButton(AlertDialog.BUTTON_POSITIVE).setTextColor(Color.BLACK);
+        d.getButton(AlertDialog.BUTTON_NEGATIVE).setTextColor(Color.BLACK);
+
+        EditText e = (EditText) d.findViewById(R.id.key_loc_name_input);
+        e.setOnFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View v, boolean hasFocus) {
+                if (hasFocus) {
+                    d.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
```
Comment (JuliusAlexanderIV) [inline app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java]: nit: isn't this line over 100 char?

**Comment by JuliusAlexanderIV** [comment]
lgtm

**Pull request merged**